modules/javafx.media/src/main/java/javafx/scene/media/MediaPlayer.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2010, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2010, 2022, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright lines must have exactly one or two years. Please remove `, 2022`

Reply:
Oh sorry, I wasn't aware of that. Now it is removed.